config/create_fortran_test.py

```diff
@@ -0,0 +1,138 @@
+#
+# Use this script to create and initial <exercise>_test.f90 file 
+# which can be used as a skeleton for your test. Typically, you will
+# have to replace 'response' in the generated file with the right 
+# function call. 
+#
+# Also note that fortran has issues with special characters such as \n and \t 
+# so take special care handling these.
+#
+# 
+# Prerequsites
```

Comment:
```suggestion
# Prerequisites
```

Reply:
Last commit from should solve issues:
- moved from config to bin
- linter errors fixed
- function doc string added